key,default_value
userAgent,"Botocore/1.16.26 Python/2.7.18 Linux/5.4.0-2-amd64"
eventID,"ccb64128-932f-45a1-97f1-6a7eebbc18b5"
errorMessage,"An unknown error occurred"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"251.105.254.1"
eventName,"ListPlatformBranches"
eventSource,"elasticbeanstalk.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"4962a9e4-5cbe-41de-ba16-f9fdbc55eba5"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2020-06-10T05:30:40Z"
